src/parsers/manifest/dash/js-parser/node_parsers/utils.ts

```diff
@@ -28,6 +28,41 @@ const iso8601Duration =
   /^P(([\d.]*)Y)?(([\d.]*)M)?(([\d.]*)D)?T?(([\d.]*)H)?(([\d.]*)M)?(([\d.]*)S)?/;
 const rangeRe = /([0-9]+)-([0-9]+)/;
 
+
+const supplementalCodecSeparator = /([,]? +)/g;
```

Comment:
I feel like `[ ]` are not needed as there's only one possibility here, the comma.
Also parentheses do not seem to be useful here as it regroups the whole regex?

Also `codec1 , codec2` would give => `codec1, , codec2`. I don't know if we want to handle that.

Just a comma is also not catched here but a comma followed by a space is, which I guess is fine as an output given by the function below but is weird in just the context of that single regex and its naming.

If comma are just normally impossible in that string but we want to handle it as if it was a potential separator, maybe we just want to return any amount of comma and spaces into a single space (`replace(/[, ]+/g, " ")`)

Reply:
> I feel like [ ] are not needed as there's only one possibility here, the comma.
Also parentheses do not seem to be useful here as it regroups the whole regex?
Also codec1 , codec2 would give => codec1, , codec2. I don't know if we want to handle that.

I agree with all this remarks, thanks, I will update.

> maybe we just want to return any amount of comma and spaces into a single space (replace(/[, ]+/g, " "))

My though was to have a unique syntax for Codecs and Supplemental codec and to replace any amount of comma and spaces into a comma and a space, (as it is the syntax used by MSE APIs)

Reply:
> For things like this function, I'm unsure if they should be declared here. The disadvantages I see is that:

I understand what you meaned now, I move this convertion logic into `parser/manifest/dash/common/utils` as it's use by both the `js-parser`and the `wasm-parser`